### PR TITLE
Update AbstractConsignment.php

### PR DESCRIPTION
**Pull request by sharkooon**
fixed when empty:
Return value of MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment::getPickupCountry() must be of the type string, null returned

### CODE REVIEW

File: src/Model/Consignment/AbstractConsignment.php [hunk 1]
```diff
@@ -1533,7 +1533,7 @@ public function addItem(MyParcelCustomsItem $item): self
     /**
      * @return string
```
Comment (RichardPerdaan) [inline src/Model/Consignment/AbstractConsignment.php:1534]:
```suggestion
     * @return string|null
```

**Review by RichardPerdaan — CHANGES_REQUESTED** [review]

**Comment by RichardPerdaan** [comment]
@sharkooon ,
Before we merge this adjustment, I would like to do a research to see why it is empty. There must always be a pickup country present.

This is why I think things are going wrong elsewhere.

**Comment by sharkooon** [comment]
Er is namelijk inconsistentie in de code wat wordt teruggegeven:
Deze returned alleen string:
```
getPickupCountry: string
getPickupLocationCode: string
```

Deze returned string of null.
```
getPickupPostalCode(): ?string
getPickupStreet(): ?string
getPickupCity(): ?string
getPickupNumber(): ?string
getPickupLocationName(): ?string
getPickupLocationCode(): string
getPickupNetworkId(): ?string
```

**Review by reindert-vetter — APPROVED** [review]

**Comment by sharkooon** [comment]
misschien dat getPickupLocationCode ook nog aangepast moet worden?

**Review by RichardPerdaan — APPROVED** [review]

**Pull request merged**